jenkins/JenkinsFile-rebuild-docker-images

```diff
@@ -173,6 +173,28 @@ pipeline {
           } // post
         } // stage
 
+        stage('ci-minimal') {
```

Comment:
Are other images going to inherit `FROM ci-minimal`?

If this dependency exist, I'm not sure the images can be built at the same time,. because it will generate a new `ci-minimal` which is not the one being used to generate the others.

To adjust to this new complexity, this pipeline should be changed to build and push `ci-minimal` to DockerHub, and only then trigger the other images build, passing a parameter that will be used to control what is the specific tag to be used in the `FROM ...` clause for all images being built from `ci-minimal`.

Reply:
I don't know of any plan to have the other images inherited from `ci-minimal`. Currently, I am unaware of any dependency between the docker images we build. Most of them inherit  `FROM ubuntu:18.04`.

Reply:
But I agree with you that having a common image inherited by others could be worth looking into. I also suspect this has been something considered in the past. Maybe @driazati knows more about this?

Reply:
There was some discussion in https://github.com/apache/tvm/pull/11809, but `ci-minimal` certainly isn't trying to be a base image for the others, just a smaller image to test tvm with fewer build options turned on than in the 25GB CPU/GPU images.